addons/mail/data/mail_templates_invite.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="mail_notification_invite" inherit_id="mail.mail_notification_layout" primary="True">
+            <xpath expr="//td[@t-if='subtitles']" position="before">
+                <t t-if="not has_button_access">
+                    <t t-set="subtitles" t-value="False" />
+                </t>
+            </xpath>
+            <xpath expr="//div[@t-out='message.body']" position="before">
+                <hr width="100%"
+    style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0;margin: 10px 0px;"/>
+            </xpath>
+            <xpath expr="//div[@t-out='message.body']" position="replace">
+                <div style="font-size:13px;">
+                    <div>
+                        <t t-out='message.author_id.name'/> (<t t-out='message.author_id.email'/>) added you as a follower of this document.
+                    </div>
+                    <div style="color:grey;">
+                        <t t-out="message.body"/>
+                    </div>
+                </div>
+            </xpath>
+            <xpath expr="//tr[td/p[@t-if='subtype_internal']]" position="replace"/>
+            <xpath expr="//span[@id='mail_unfollow']" position="replace"/>
+            <xpath expr="//div[@style='margin-top:16px;']/hr" position="before">
+                <div valign="center">
+                    <p t-if="subtype_internal" style="font-size: 13px;">
+                        Replying will post an internal note. Followers won't receive any email notification.
+                    </p>
+                    <p t-if="subtype and subtype.default" style="font-size: 13px;">
+                        Reply to this email to join the conversation.
+                    </p>
```

Comment:
as reply-to is globally user's email, I fear it won't work. Not sure what we expect when answering invite emails, but I am unsure it should be considered as a reply to the thread.

Reply:
According to my understanding we want reply-to to be counted as communication. Not sure, I may be missing something :thinking: